src/app/server/server.js

```diff
@@ -26,7 +27,12 @@ module.exports = function (options) {
   app.use(
     function setup(req, res, next) {
       req.v4app = this;
-      req.sandboxCredentials = options.sandboxCredentials || {};
+      req.sandboxCredentials = req.sandboxCredentials || {};
```

Comment:
Do we have docs to mention setting the sandboxCredentials via `req` rather than through `options`?